src/game/Game.tsx

```diff
@@ -0,0 +1,92 @@
+import type { Socket } from 'socket.io-client';
+import type { FormEvent } from 'react';
+
+import React from 'react';
+import { css, StyleSheet } from 'aphrodite';
+import Player from './Player';
+import ConnectionStatus from './ConnectionStatus';
+
+type Props = Readonly<{
+  players: Array<any>;
+  viewerID: string;
+  lobbyID: string;
+  socket: Socket;
+  hostID: string;
+}>;
+
+function Game({
+  socket,
+  viewerID,
+  players,
+  lobbyID,
+  hostID,
+}: Props): React.ReactElement {
+  function onReset(e: FormEvent<HTMLButtonElement>): void {
+    e.preventDefault();
+    socket.emit('reset_game', { lobbyID });

```

Comment:
Same as above. just highlighting all of these for now

Reply:
Replied above